### DISCUSSION TITLE
Graphics images not being created in one book chapter

### DISCUSSION THREAD

**Discussion by harrelfe** [quarto-dev/quarto-cli#1310] [Q&A] [answered]
I moved a book to a new directory, removed _freeze, re-started a book project, and rendered all chapters.   All chapters look fine except for one, using `Quarto` 0.9.648.  The directory `_book/chaptername_files` is empty even though there were no execution errors.

I haven't changed any of the chapter script.  I'm sorry I can't produce a short example.

**UPDATE** The chapter starts with two `mermaid` charts.  When I remove those two chunks the graphs render as they are supposed to.

**UPDATE 2** When I put the chapter as a self-contained article there is no problem with the figures.  The `book` environment seems to have a problem.

Here is a minimal failing example book: https://hbiostat.org/attach/btest.zip

**UPDATE 3** I had to go all the way back to `Quarto` 0.9.560 to get the figures rendered when preceded by a `mermaid` chunk.

**Comment by harrelfe** [comment]
I'm a little desperate on this one if someone can come up with a fix.  The older version of `Quarto` I'm having to use to render plots in the same chapter in which there is a `mermaid` script is having some clashing of marginal notes that I think was fixed in a later release.

**Comment by jjallaire** [comment, marked as answer]
Created an issue for this here: https://github.com/quarto-dev/quarto-cli/issues/1340

We will fix this ASAP.
